doc/source/user/absolute_beginners.rst

```diff
@@ -5,212 +5,195 @@ NumPy: the absolute basics for beginners
 
-One way we can initialize NumPy arrays is from Python lists, using nested lists
-for two- or higher-dimensional data.
+Elements of an array can be accessed in :ref:`various ways
+<quickstart.indexing-slicing-and-iterating>`. For instance, we can access an
+individual element of this array as we would access an element in the original
+list: using the integer index of the element within square brackets.
 
-For example::
+    >>> a[0]
+    1
 
-  >>> a = np.array([1, 2, 3, 4, 5, 6])
+.. note::
 
-or::
+     As with built-in Python sequences, NumPy arrays are "0-indexed": the first
+     element of the array is accessed using index ``0``, not ``1``.
 
-  >>> a = np.array([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+As with the original list, Python slice notation can be used for indexing.
 
-We can access the elements in the array using square brackets. When you're
-accessing elements, remember that indexing in NumPy starts at 0. That means that
-if you want to access the first element in your array, you'll be accessing
-element "0".
+    >>> a[:3]
+    array([1, 2, 3])
 
-::
+Also like the original list, the array is mutable.
 
-  >>> print(a[0])
-  [1 2 3 4]
+    >>> a[0] = 10
+    >>> a
+    array([10,  2,  3,  4,  5,  6])
 
+One major difference between the array and the original list is that slicing
+returns a *view*: an object that refers to the data in the original array. The
+original array can be mutated using the view.
 
-More information about arrays
------------------------------
+    >>> b = a[3:]
+    >>> b
+    array([4, 5, 6])
+    >>> b[0] = 40
+    >>> a
+    array([ 1,  2,  3, 40,  5,  6])
 
-*This section covers* ``1D array``, ``2D array``, ``ndarray``, ``vector``, ``matrix``
+Two- and higher-dimensional arrays can be initialized from nested Python
+sequences::
 
-------
+    >>> a = np.array([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+    >>> a
+    array([[ 1,  2,  3,  4],
+           [ 5,  6,  7,  8],
+           [ 9, 10, 11, 12]])
 
-You might occasionally hear an array referred to as a "ndarray," which is
-shorthand for "N-dimensional array." An N-dimensional array is simply an array
-with any number of dimensions. You might also hear **1-D**, or one-dimensional
-array, **2-D**, or two-dimensional array, and so on. The NumPy ``ndarray`` class
-is used to represent both matrices and vectors. A **vector** is an array with a
-single dimension (there's no difference
-between row and column vectors), while a **matrix** refers to an
-array with two dimensions. For **3-D** or higher dimensional arrays, the term
-**tensor** is also commonly used.
+In NumPy, a dimension of an array is sometimes referred to as an "axis". This
+terminology may be useful to disambiguate between the dimensionality of an
+array and the dimensionality of the data represented by the array. For
+instance, the array ``a`` could represent three points, each lying within a
+four-dimensional space, but ``a`` has only two "axes".
```

Comment:
Maybe keep the note about the "ndarray" name somewhere?

Reply:
How about in the line:
> An "array" is an instance of the fundamental data structure class of the NumPy
library, the ``ndarray``. A common way to think of an array is as a grid of elements.

Reply:
@melissawm Actually, I took another crack at this: how does https://github.com/numpy/numpy/pull/24981#discussion_r1372327297 look to you?